docs/tutorials/services/webcam-line-follower-robot.md

```diff
@@ -40,7 +40,7 @@ This tutorial uses a standard webcam in place of these sensors, and allows a rob
 
 If you'd like to directly see the code, check out the [**Line Follower Code on GitHub**](https://github.com/viam-labs/line-follower/).
 
-{{<imgproc src="/tutorials/webcam-line-follower/lf-scuttle2.png" resize="600x" declaredimensions=true alt="A SCUTTLE Robot base with a camera mounted on the front, pointing mostly down and slightly forwards.">}}
+{{<imgproc src="/tutorials/webcam-line-follower/lf-scuttle2.png" resize="600x" declaredimensions=true alt="A Scuttle robot base with a camera mounted on the front, pointing mostly down and slightly forwards.">}}
```

Comment:
Are these all because the screen reader is trying to spell it out?

Reply:
yes